templates/selenium-python37/src/__ALGO___test.py

```diff
@@ -0,0 +1,8 @@
+from __ALGO__ import apply
+
+def test_algorithm():
+    input = {}
+    result = apply(input)
+    assert result == {"status": "success"}
+
+test_algorithm()
```

Comment:
You shouldn't need this line. normal pytest will run all methods that start with "test"
Example: https://github.com/algorithmiaio/langpacks/blob/master/python/template/src/__ALGO___test.py

(Also, testing in IPA doesn't work... just saying)